packages/core/src/metadata.ts

```diff
@@ -73,6 +166,13 @@ class Metadata implements IMetadata {
     };
   };
 
+  tss?: {
+    secp256k1: TssMetadata;
+    ed25519: TssMetadata;
```

Comment:
Make this a Record of `KeyType` and `TssMetadata` instead of fixing the entries here?